frontend/packages/dev-console/src/utils/resource-label-utils.ts

```diff
@@ -45,6 +45,15 @@ export const getCommonAnnotations = () => {
   };
 };
 
+export const getTriggerAnnotation = (name: string) => ({
+  'image.openshift.io/triggers': JSON.stringify([
```

Comment:
import { TRIGGERS_ANNOTATION } from '@console/shared';
```suggestion
  [TRIGGERS_ANNOTATION]: JSON.stringify([
```